src/binary-reader-objdump.h

```diff
@@ -37,18 +47,21 @@ typedef struct WasmObjdumpOptions {
   WasmBool raw;
   WasmBool disassemble;
   WasmBool debug;
+  WasmBool relocs;
   WasmObjdumpMode mode;
   const char* infile;
   const char* section_name;
   WasmBool print_header;
+  WasmStringSliceVector function_names;
```

Comment:
its because it needs to persist between passes.  I though about creating another "state/context" object that could be passed in along with options.